app/web_framework/cluster_base_handler.py

```diff
@@ -62,6 +64,31 @@ def _handle_request_exception(self, ex):
         self.set_status(status_code)
         self.finish()
 
+    # For methods that annotate with `Tuple`, use `disable=invalid-sequence-index`
+    # This bug is fixed in https://github.com/PyCQA/astroid/commit/563031aaf13a44adc8db4f8d0ab8020d550aae00
+    # More information on the issue in https://github.com/PyCQA/pylint/issues/1212
+
+    def get_pagination_params(self) -> Tuple[int, int]:  # pylint: disable=invalid-sequence-index
+        offset = self.get_query_argument('offset', Configuration['pagination_offset'], True)
+        limit = self.get_query_argument('limit', Configuration['pagination_limit'], True)
+        try:
+            offset = int(offset)
+            limit = int(limit)
+        except ValueError as ex:
+            raise BadRequestError from ex
+        return self._validate_pagination_params(offset, limit)
+
+    @staticmethod
+    def _validate_pagination_params(offset: int, limit: int) -> Tuple[int, int]:  # pylint: disable=invalid-sequence-index
```

Comment:
You're right, this could do with a better name.

I intentionally separated the two methods into Tornado retrieving the values and then sanitizing the values. This way we can test the sanitizing the values function since there isn't a good way to test the former.